internal/pkg/template/templates/workloads/partials/cf/subscribe.yml

```diff
@@ -80,16 +80,31 @@ EventsQueue:
       deadLetterTargetArn: !GetAtt DeadLetterQueue.Arn
       maxReceiveCount: {{.Subscribe.Queue.DeadLetter.Tries}}
     {{- end}}
+    {{- if eq (deref .Subscribe.Queue.Type) "fifo"}}
+    FifoQueue: true
+    {{- end}}
+    {{- if .Subscribe.Queue.FifoThroughputLimit }}
+    FifoThroughputLimit: {{.Subscribe.Queue.FifoThroughputLimit}}
+    {{- end}}
+    {{- if .Subscribe.Queue.DeduplicationScope }}
+    DeduplicationScope: {{.Subscribe.Queue.DeduplicationScope}}
+    {{- end }}
+    {{- if .Subscribe.Queue.ContentBasedDeduplication }}
+    ContentBasedDeduplication: {{.Subscribe.Queue.ContentBasedDeduplication}}
+    {{- end}}
   {{- end}}
 
 {{- if .Subscribe.Queue}}{{- if .Subscribe.Queue.DeadLetter}}
 DeadLetterQueue:
```

Comment:
huh TIL I didn't know that the dead letter queue of a FIFO queue must also be FIFO:
https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html

This is a really nice catch 👍

Reply:
To add to your point, I was wondering if we should be allowing customers to set fifo related configs to Deadletter queue as well? It will be a lot of configurations that we will be exposing to the customer at once. I think we can add those configs if customers ask for it. What do you think about it?

Reply:
>It will be a lot of configurations that we will be exposing to the customer at once. I think we can add those configs if customers ask for it. What do you think about it?

Sounds good!